plugins/testPlugin/testPlugin.cpp

```diff
@@ -422,11 +429,12 @@ void mumble_onKeyEvent(uint32_t keyCode, bool wasPress) {
 
 bool mumble_hasUpdate() {
 	// This plugin never has an update
```

Comment:
code comment is outdated

Reply:
It was for a short time in between, but the respective change got reverted again and the function should now always return `false` again...